core/src/ast/mod.rs

```diff
@@ -99,6 +99,7 @@ pub enum Statement {
     /// SHOW VARIABLE
     #[cfg(feature = "metadata")]
     ShowVariable(Variable),
+    ShowIndexes(ObjectName),
```

Comment:
Sorry for the late review.
I was thinking about to provide you a good one shot review, but it becomes too delayed.

Here, `SHOW INDEXES - ` only works for storages which has `Index` support.
```rust
#[cfg(feature = "index")]
CreateIndex,             
#[cfg(feature = "index")]
DropIndex,               
#[cfg(feature = "index")]
ShowIndexes(ObjectName), 
```

It would be good to handle `Payload::ShowIndexes(_)` as optional feature.